app/helpers/parse/toNumber.spec.js

```diff
@@ -0,0 +1,31 @@
+const assert = require('assert')
+
+const toNumber = require('./toNumber')
+
+describe('toNumber', function () {
+    describe('when value already is number', function () {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/UFABCNextOps/ufabc-next-server/pullRequest?prid=8443142)
```suggestion
  describe('when value already is number', function () {
```